src/backend/joanie/core/migrations/0001_initial.py

```diff
@@ -218,11 +215,7 @@ class Migration(migrations.Migration):
                 (
                     "resource_link",
                     models.CharField(
-                        blank=True,
-                        max_length=200,
-                        null=True,
-                        unique=True,
-                        verbose_name="resource link",
+                        max_length=200, unique=True, verbose_name="resource link"
```

Comment:
I did not find this change in the model?
It is normally done in my latest PR?

Reply:
Nope, I missed that during the review. In our latest PR, initial migration did not reflect this change so I included it here.

Reply:
oh! ok :pray: